codewell-fe/src/Components/User/Login.js

```diff
@@ -1,12 +1,18 @@
 import React, { useState } from 'react';
 
+import { useLocation } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 import { Alert, Button, Card, Form, Input, Layout, Space } from 'antd';
 import { UserOutlined, LockOutlined } from '@ant-design/icons';
 
+import { getInitialAlertDescription } from '../../Helpers/Login.js';
+
 function Login(props) {
   const { t } = useTranslation();
-  const [error, setError] = useState('');
+  const location = useLocation();
+  const [alertDescription, setAlertDescription]
+    = useState(getInitialAlertDescription(location));
```

Comment:
I made a mistake here, equality stays on the same row. sorry for this
```suggestion
  const [alertDescription, setAlertDescription] =
    useState(getInitialAlertDescription(location));
```